docs/source/documentation/orchestrator/index.rst

```diff
@@ -2,12 +2,32 @@
 Orchestrator
 ************
 
-The Orchestrator has two functions.
-Its first role is to handle the ledger containing all known assets.
-It also dispatches tasks to the relevant :term:`Organizations<Organization>`.
+Performing a Federated Learning experiment implies a lot of different compute tasks: local training, aggregation, testing on different organizations, etc. The role of the orchestrator is to distribute ML tasks among organizations, while ensuring complete traceability of operations.
+
+The orchestrator registers the status of tasks; when a task is done, it evaluates if some tasks in ``Waiting`` are now unblocked, and if it's the case, the status of those tasks is changed to ``To do``. The new status is sent to all the backends, who store the new tasks ``To do`` in the task queue (Celery). Then, the task queue will assign the task to one of the workers (if multiple) and handle retries if needed.
+
+
+Centralized vs. decentralized orchestration
+===========================================
+
+Substra offers two types of orchestration: distributed and centralized.
+
+.. image:: /static/schemes/distributed-vs-centralized-orc.png
```

Comment:
The scheme background is blank, you need to make it transparent (or it'll be weird on the doc page)